lib/beaker-pe/pe-client-tools/executable_helper.rb

```diff
@@ -75,7 +75,9 @@ def login_with_puppet_access_on(host, credentialed_dispatcher, opts={})
 
             user = credentialed_dispatcher.credentials.login
             password = credentialed_dispatcher.credentials.password
-            puppet_access_on(host, 'login', {:stdin => "#{user}\n#{password}\n"})
+            args = ['login']
+            args.push "--lifetime #{lifetime}" if lifetime
```

Comment:
Nick might've been asking if `--lifetime` and `#{lifetime}` need to be separate entries in the array.